src/tests/verifyAccount.test.js

```diff
@@ -30,7 +30,6 @@ describe('VERIFICATION TESTS', () => {
       .post('/api/v1/auth/register')
       .send(newUser)
       .end((err, res) => {
-        // const { token } = res.body;
```

Comment:
hey @Ntare22 could you remove commented codes on this line?

Reply:
They were removed... that is why it's red.